### PR TITLE
[FROGGER-2] Created class and methods

### PR DESCRIPTION
**Pull request by szyles**
Please review

### CODE REVIEW

**Comment by AdZawadzki** [comment]
NICEE

File: frogger.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class Frogger{
+
```
Comment (teclamat) [inline frogger.js:3]: Something's missing here, maybe... `constructor()`?

**Review by teclamat — CHANGES_REQUESTED** [review]

File: frogger.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class Frogger{
```
Comment (teclamat) [inline frogger.js]: Missing space before braces

**Review by teclamat — CHANGES_REQUESTED** [review]

**Comment by szyles** [comment]
Please review once more time

**Review by teclamat — APPROVED** [review]

**Pull request merged**